deployments/ansible/roles/signalfx-agent/tasks/yum_repo.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Add Signalfx Agent repo into source list
+  yum_repository:
+    name: signalfx-agent
+    description: SignalFx Agent Repository
+    baseurl: "{{ rhel_repo_url }}"
+    gpgkey: "{{ rhel_gpg_key_url }}"
+    gpgcheck: yes
+    enabled: yes
+
+- name: Install Signalfx Agent via yum package manager
+  yum:
+    name: signalfx-agent
+    state: present
+
+- name: add signalfx group
+  group:
+    name: signalfx-agent
+    state: present
+
+- name: add signalfx user
+  user:
+    name: signalfx-agent
+    state: present
+
+- name: write signalfx config
```

Comment:
This file should have user/group of `signalfx-agent`.

Reply:
done